run_dir/static/js/project_samples.js

```diff
@@ -319,10 +319,14 @@ function load_all_udfs(){
 
       // Set the project name and status
       if (prettify(key) == 'project_name'){
+        let priority_badge = ""
+        if(data['priority']==="High"){
+          priority_badge = "<span class='badge bg-imp'> Priority <span class='fa-solid fa-flag'></span></span>"
+        }
         if (!data['portal_id']) {
-          project_title = project + ", " + data['project_name'] + " <small class='text-muted'>(" + data['customer_project_reference'] + " - no order in NGI portal)</small>";
+          project_title = project + ", " + data['project_name'] + " <small class='text-muted'>"+ priority_badge+" (" + data['customer_project_reference'] + " - no order in NGI portal)</small>";
```

Comment:
Great! But could we maybe have the badge on the other side of the project name? 

Either before or after the status badge.


![Skärmavbild 2023-06-29 kl  11 31 37](https://github.com/SciLifeLab/genomics-status/assets/37932118/21cf13b1-3056-44e5-9bce-3caeca419b1d)

Should the red color be a bit brighter? 🤔

Reply:
The descriptions can be of varying lengths, so it can be easy to miss if its at the end.

Reply:
Ah, I see. The danger color looks better! Could we also add `mr-2` to the badge? :smile: